src/stylesheets/components/_card.scss

```diff
@@ -191,8 +191,12 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   &--vertical {
 
     @include at-media('mobile') {
```

Comment:
Do we still need mobile and tablet breakpoint if all sizes should have the header on the left?

Reply:
We can keep the tablet breakpoint in case you want any changes for tablet.